be/src/exec/vectorized/olap_scan_node.cpp

```diff
@@ -562,8 +567,11 @@ Status OlapScanNode::_capture_tablet_rowsets() {
             std::shared_lock l(tablet->get_header_lock());
             RETURN_IF_ERROR(tablet->capture_consistent_rowsets(Version(0, version), &_tablet_rowsets[i]));
         }
+        StringAppendF(&_opened_tablet_ids, "%ld,", tablet_id);
+    }
+    if (!_opened_tablet_ids.empty()) {
+        LOG(INFO) << "open rowsets for " << _opened_tablet_ids;
```

Comment:
This will cause too many logs, better to use VLOG

Reply:
will not merge, just run in daily